proto/hummock.proto

```diff
@@ -406,6 +428,8 @@ service HummockManagerService {
   rpc TriggerManualCompaction(TriggerManualCompactionRequest) returns (TriggerManualCompactionResponse);
   rpc ReportFullScanTask(ReportFullScanTaskRequest) returns (ReportFullScanTaskResponse);
   rpc TriggerFullGC(TriggerFullGCRequest) returns (TriggerFullGCResponse);
+  rpc GetPinnedVersionsSummary(GetPinnedVersionsSummaryRequest) returns (GetPinnedVersionsSummaryResponse);
```

Comment:
I'd prefer prefixing APIs solely used by risectl with RiseCtl. e.g., `RiseCtlGetPinnedVersionsSummary`, so that developers won't use this API in somewhere else.